### PR TITLE
Add further successful tweaks and minification steps

### PR DESCRIPTION
**Pull request by henryefranks**
Note the careful use of the word 'successful' in the title there: this is everything I had in my notebook that actually worked in the end. We could add a guide to how repositioning *could* work but currently doesn't, but I don't think it's worth the effort. Just give these a once over to check it all reads ok and there aren't any mistakes/typos.

### CODE REVIEW

**Review by eleanor-clifford — CHANGES_REQUESTED** [review]
Any chance we can also get instructions for the tweaks? Nw if it will be a ton
of effort

**Review by eleanor-clifford — CHANGES_REQUESTED** [review]
Lol I never submitted this review, whoops

File: docs/minify.md [hunk 1]
```diff
@@ -1,22 +1,95 @@
...
+
+QEMU provides a tool for resizing a qcow2 image, `qemu-img resize`. To shrink
+the size of the disk (make sure not to make it smaller than the Windows
+partition!), use `qemu-img resize --shrink img.qcow2 SIZE` (see `man qemu-img`
+for SIZE).
+
+
+### Sparsify and Compress VM Image
+
+An important step in minimising image size is to sparsiy it. Optionally, you
```
Comment (tim-clifford) [inline docs/minify.md:84]:
```suggestion
An important step in minimising image size is to sparsify it. Optionally, you
```
smh spell right

File: docs/suggested-tweaks.md [hunk 2]
```diff
@@ -1,15 +1,133 @@
...
-- Set this wallpaper
-![linus](https://i.imgur.com/LwYlpqn.png)
+# Suggested Tweaks in Windows for a Better User Experience
+
+### Run the Windows 10 Debloater
+
+To easily remove much of the pre-packaged bloatware in Windows 10, you can use
+[this debloating script](https://github.com/Sycnex/Windows10Debloater). A full
+guide to using the script can be found in the repo, but if you're unsure, you
+can't go wrong with the GUI.
```
Comment (tim-clifford) [inline docs/suggested-tweaks.md:8]:
```suggestion
can't go wrong with the GUI. Except maybe bricking your system. Probably not
though.
```

**Review by eleanor-clifford — CHANGES_REQUESTED** [review]
Looks great. just a couple things as usual

**Review by eleanor-clifford — APPROVED** [review]
LGTM, merging

**Pull request merged**